src/queries/getAllBusinesses.js

```diff
@@ -0,0 +1,7 @@
+const db = require("../database/db_connection");
+
+const getAllBussiness = data => {
```

Comment:
lets make this function consistent, and have the arrow function do the return instead of the return keyword

Reply:
fixed